protobuf/module_msg.proto

```diff
@@ -87,6 +87,20 @@ message MeasureCommandGetSummaryResponse {
   uint64 latency_99_ns = 9;
 }
 
+message DRRArg {
+  uint32 num_flows = 1;//Number of flows to handle in module
```

Comment:
See the correct comment style in this pull request:

https://github.com/NetSys/bess/pull/304